src/components/Journal/FinishJournal.jsx

```diff
@@ -181,7 +182,26 @@ const FinishJournal = props => {
             });
         } catch (e) {
             if (networkFailure(e.message)) {
-                toast(`We can't reach our servers. Check your internet connection and try again.${props.files.length > 0 ? " Your images might also be too big." : ""}`);
+                ldb.logs.add({
+                    timestamp: DateTime.local().toMillis(),
+                    year: DateTime.local().year,
+                    month: DateTime.local().month,
+                    day: DateTime.local().day,
+                    time: DateTime.local().toFormat("h:mm a"),
+                    zone: DateTime.local().zoneName,
+                    mood: props.moodWrite,
+                    journal: props.text,
+                    average: props.average,
+                    audio: audioBlob,
```

Comment:
putting huge blob in indexeddb is kinda crazy, reminder for me to test this on mobile to make sure it doesn't destroy the device

Reply:
k :thumbs: